app/controllers/delivery_zones_controller.rb

```diff
@@ -0,0 +1,29 @@
+class DeliveryZonesController < ApplicationController
+  def index
+    @delivery_zones = DeliveryZone.all
+  end
+
+  def show
+    @delivery_zone = DeliveryZone.find_by!(zipcode: params[:id])
+  end
+
+  def create
+    @delivery_zone = DeliveryZone.new(delivery_zone_params)
+
+    if @delivery_zone.save
+      redirect_to delivery_zone_url(@delivery_zone)
```

Comment:
Is there a reason that you are using the `_url` path helper here? Related question: If the answer to the previous question is "No", then what are your thoughts on just using `redirect_to @delivery_zone` here?